modules/reliability/jsx/columnFormatter.js

```diff
@@ -0,0 +1,27 @@
+function formatColumn(column, cell, rowData, rowHeaders) {
+// If a column if set as hidden, don't display it
+  if (loris.hiddenHeaders.indexOf(column) > -1) {
+    return null;
+  }
+
+  // Create the mapping between rowHeaders and rowData in a row object.
+  var row = {};
+  rowHeaders.forEach(function(header, index) {
+    row[header] = rowData[index];
+}, this);
+
+ 
+    if(column === 'PSCID' ) {
+        if (row["Manual Swap"]==="yes")
+        {
```

Comment:
```js
if (stmnt)
{
 // stuff
}
```
@driusan This is valid format, but as you mentioned not standard in Loris and hard to read. ESLint would definetly be apreciated.